docs/compliance/fedramp.md

```diff
@@ -4,8 +4,33 @@ keywords: standards, compliance, security
+The [Federal Risk and Authorization Management Program
+(FedRAMP)](https://www.fedramp.gov/) is a U.S. Federal Government-wide program
+that enables a standardized approach to security assessment and authorization.
+Federal agencies that choose to leverage cloud services must ensure that they're
+utilizing FedRAMP-authorized providers. The security controls FedRAMP requires a
+provider to adhere to are a subset of the controls documented by NIST Special
+Publication 800-53. As with the baselines set within NIST 800-53 (low, moderate
+and high), FedRAMP also incorporates these same baselines in its authorization
+process. In addition, when agencies deploy systems (like Docker Enterprise
+Edition) on top of these providers, they must acquire an Authority to Operate
+(ATO) for those system that are in line with security procedures for those
+agencies.
 
-It is important to note that Docker, Inc is not a cloud service provider. While Docker does offer various SaaS-hosted services, which include Docker Hub, Docker Store and Docker Cloud, these services are *not* FedRAMP provisionally authorized by the Joint Authorization Board (JAB) nor agency-authorized. However, Docker's Enterprise product stack can be installed on top of compute services offered by a number of FedRAMP provisionally-authorized infrastructure-as-a-service (IaaS) providers. Examples include Microsoft Azure Government and Amazon Web Services GovCloud. Federal agencies can subsequently inherit the FedRAMP controls already satisfied by those providers and can combine those controls with the NIST 800-53 controls applicable to Docker Enterprise Edition and that which are documented on our site in order to streamline your agency's ability to get an authority to operate (ATO) for Docker Enterprise Edition.
+It is important to note that Docker, Inc is not a cloud service provider. While
+Docker does offer various SaaS-hosted services, which include Docker Hub, Docker
+Store and Docker Cloud, these services are *not* FedRAMP provisionally
+authorized by the Joint Authorization Board (JAB) nor agency-authorized.
+However, Docker's Enterprise product stack can be installed on top of compute
+services offered by a number of FedRAMP provisionally-authorized
+infrastructure-as-a-service (IaaS) providers. Examples include Microsoft Azure
+Government and Amazon Web Services GovCloud. Federal agencies can subsequently
+inherit the FedRAMP controls already satisfied by those providers and can
+combine those controls with the NIST 800-53 controls applicable to Docker
+Enterprise Edition and that which are documented on our site in order to
```

Comment:
full-stop after Enterprise Edition. I don't understand where you were going the rest of the sentence but you should reword that to be less of a run-on.

Please remove the phrase "in order to" from your vocabulary. ;)